x-pack/plugins/observability/server/assets/enrich_policies/slo_summary_enrich_policy.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SO_SLO_TYPE } from '../../saved_objects';
+import { SLO_SUMMARY_ENRICH_POLICY_NAME } from '../constants';
+
+export const getSLOSummaryEnrichPolicy = () => ({
+  name: SLO_SUMMARY_ENRICH_POLICY_NAME,
+  match: {
+    indices: '.kibana',
```

Comment:
@kdelemme I am trying to understand how things work and where data is stored. What gets stored in this index `.kibana`? Saved objects?

Doing `GET .kibana/_search`, I would expect to see some entry of `"type": "slo"`, but no nothing related to slo in the kibana index. How can I access/see the SLO definition of the Saved Object?

Reply:
Ok I think I found something using the [find objects api](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-find.html)

`GET kbn:api/saved_objects/_find?type=slo` I was able to get the 2 slo definitions of my 2 SLOs.

 But could you please clarify what this `.kibana` index used for and how I can access some data (request I can make on Dev Tools to get some slo related data on the kibana index)?
`

Reply:
```
POST .kibana_*/_search
{
  "size": 10, 
  "query": {
    "bool": {
      "filter": [
        {
          "term": {
            "type": "slo"
          }
        }
      ]
    }
  }
}
```

Reply:
Ok I think I got it. Using this, returned me the data I wanted to see:

```
GET .kibana/_search
{
  "query": {
    "bool": {
      "filter": [
        {
          "match_phrase": {
            "type": "slo"
          }
        }
      ]
    }
  }
}
```

Do you know why the `slo` type was not returned with a simple `GET .kibana/_search` or even `GET .kibana*/_search`